guestbook/register/forms.py

```diff
@@ -0,0 +1,44 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import validate_email
+
+
+
+class ValidUniqueUserEmailField(forms.EmailField):
```

Comment:
From this I guess you're trying to change the behavior of the built-in user model. Have a look at https://docs.djangoproject.com/en/1.8/topics/auth/customizing/#substituting-a-custom-user-model on how to do it in a much more flexible way :wink: